src/constants/poi.data.ts

```diff
@@ -1459,5 +1494,408 @@ export const POIInfo: POI[] = [
+
+    description: "",
+    location: {
+      latitude: 45.458174,
+      longitude: -73.640072
+    },
+    buildingId: BuildingId.CC,
+    level: 1,
+    category: POICategory.Classroom
+  },
+  {
+    id: null,
+    displayName: "104",
+
+    description: "",
+    location: {
+      latitude: 45.458182,
+      longitude: -73.639984
+    },
+    buildingId: BuildingId.CC,
+    level: 1,
+    category: POICategory.Classroom
+  },
+  // ------------------
+  // SGW JMSB
+  // ------------------
+  {
+    id: null,
+    displayName: "Mainsonneuve Exist",
+
+    description: "Exit/Entrance to JMSB and the corner of dM and Guy",
+    location: {
+      latitude: 45.49554740223838,
+      longitude: -73.57927136202986
+    },
+    buildingId: BuildingId.MB,
+    level: 1,
+    category: POICategory.Exit
+  },
+  {
+    id: null,
+    displayName: "Police St. Exist",
+
+    description: "Exit/Entrance to JMSB and the corner of Police St. and Guy",
+    location: {
+      latitude: 45.495216991097614,
+      longitude: -73.5785603480603
+    },
+    buildingId: BuildingId.MB,
+    level: 1,
+    category: POICategory.Exit
+  },
+  {
+    id: null,
+    displayName: "Stairs dM",
+
+    description: "non-accessible stairs that can only go to higher floors",
+    location: {
+      latitude: 45.49533747193755,
+      longitude: -73.57944214004628
+    },
+    buildingId: BuildingId.MB,
+    level: 1,
+    category: POICategory.Stairs
+  },
+  {
+    id: null,
+    displayName: "Men's",
+
+    description: "JMBS first 1st floor men's washroom",
+    location: {
+      latitude: 45.49533465172625,
+      longitude: -73.57934289831273
+    },
+    buildingId: BuildingId.MB,
+    level: 1,
+    category: POICategory.Washroom
+  },
+  {
+    id: null,
+    displayName: "Women's",
+
+    description: "JMBS first 1st floor women's washroom",
+    location: {
+      latitude: 45.49536097369291,
+      longitude: -73.5793187584316
+    },
+    buildingId: BuildingId.MB,
+    level: 1,
+    category: POICategory.Washroom
+  },
+  {
+    id: null,
+    displayName: "Security",
+
+    description: "Lost and founc and other related security services",
+    location: {
+      latitude: 45.4954328890034,
+      longitude: -73.57924164492242
+    },
+    buildingId: BuildingId.MB,
+    level: 1,
+    category: POICategory.Service
+  },
+  {
+    id: null,
+    displayName: "Elevators",
+
+    description: "Fist 1st floor elevators",
+    location: {
+      latitude: 45.49529657885985,
+      longitude: -73.57910082894914
+    },
+    buildingId: BuildingId.MB,
+    level: 1,
+    category: POICategory.Elevator
+  },
+  {
+    id: null,
+    displayName: "Escalators",
+
+    description: "Escalators that leads to tunnel",
+    location: {
+      latitude: 45.49547821216783,
+      longitude: -73.57911271713982
+    },
+    buildingId: BuildingId.MB,
+    level: 1,
+    category: POICategory.Escalator
+  },
+  {
+    id: null,
+    displayName: "Stars to second floor",
```

Comment:
Typo for stairs

Reply:
@osahtout @cindyslittleplanet @ys-lin Can you fix the typos in the displaynames for the POIs? and also make them shorter? If not I'll tackle it later today

Reply:
Yes, I'm gonna go over the POIs